### PR TITLE
add nader-ziada as reviewer for serving and scaling

### PR DESCRIPTION
**Pull request by nader-ziada**
# Changes

- add `nader-ziada` as reviewer for serving and scaling

### CODE REVIEW

File: peribolos/knative-sandbox.yaml
Comment (markusthoemmes) [inline peribolos/knative-sandbox.yaml:1135]: /assign @maximilien @zhanggbj

Comment (nader-ziada) [reply peribolos/knative-sandbox.yaml:1135]: my goal with being added to the kperf repo is to make it easier to get my contributions merged in as well as working on getting more folks contributing as well

Comment (maximilien) [reply peribolos/knative-sandbox.yaml:1135]: Why not be a reviewer first then we can add you as committer when you have some contributions merged.

Super happy to have you on board BTW.

**Review by maximilien — CHANGES_REQUESTED** [review]
Left comment. Let’s do reviewer and then committer later. 

Cc: @zhanggbj WDYT?

File: peribolos/knative-sandbox.yaml
Comment (nader-ziada) [reply peribolos/knative-sandbox.yaml:1135]: there was no kperf reviewers group to add my self to, but happy to remove myself from this for now until I have more PRs merged in. 
Already have 2 merged in and 1 in progress

Comment (nader-ziada) [reply peribolos/knative-sandbox.yaml:1135]: but I please ask the maintainers of the kperf repo to at least merge in dependencies PRs and such so it doesn't block progress

**Review by markusthoemmes — APPROVED** [review]
/lgtm
/approve

**Comment by nader-ziada** [comment]
@markusthoemmes I updated the commit message as you were adding the lgtm

**Review by julz — APPROVED** [review]
/lgtm

**Pull request merged**
